### PR TITLE
added AJAX to API

### PR DESCRIPTION
**Pull request by ks603**
Ajax page please look!

### CODE REVIEW

File: assets/scripts/books/api.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const getBooks = function () {
+  return $.ajax({
+    url: config.apiUrl + '/books',
```
Comment (JakeSeib) [inline assets/scripts/books/api.js:7]: config is an undeclared variable, I think you meant to import scripts/config.js

**Review by JakeSeib — CHANGES_REQUESTED** [review]

File: assets/scripts/books/api.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    method: 'GET',
+    headers: {
+      Authorization: 'Token token=' + store.user.token
+    }
```
Comment (srsexton94) [inline assets/scripts/books/api.js]: So I think we want to remove this as we don't have any authorization/authentication

File: assets/scripts/books/api.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const getBooks = function () {
+  return $.ajax({
+    url: config.apiUrl + '/books',
+    method: 'GET',
+    headers: {
+      Authorization: 'Token token=' + store.user.token
```
Comment (JakeSeib) [inline assets/scripts/books/api.js]: Should import store too

**Review by srsexton94 — CHANGES_REQUESTED** [review]
from comments:
So I think we want to remove this as we don't have any authorization/authentication

**Review by JakeSeib — APPROVED** [review]

**Review by srsexton94 — APPROVED** [review]
Yay!

**Pull request merged**